include/fiction/algorithms/simulation/sidb/convert_potential_to_distance.hpp

```diff
@@ -0,0 +1,57 @@
+//
+// Created by Jan Drewniok on 10.11.23.
+//
+
+#ifndef FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+#define FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+
+#include "fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp"
+#include "fiction/technology/physical_constants.hpp"
+
+#include <cmath>
+#include <cstdint>
+#include <limits>
+#include <vector>
+
+namespace fiction
+{
+/**
+ * This function converts a given electrostatic potential to the corresponding distance for given
```

Comment:
Is it a local potential or a potential between SiDBs?

Reply:
I hope the new docstring is clearer.